new_core_concepts/modules/ROOT/pages/typeql/query-variables-patterns.adoc

```diff
@@ -195,8 +195,8 @@ Finished. Total rows: 2
 
 === Negations
 A pattern is negated by wrapping it in a `not` block.
-An answer "satisfies" the negation iff it cannot be (part of) an answer to the negated pattern.
-Hence, answers are 'eliminated' by negations if the pattern inside is matched.
+An answer "satisfies" the negation if it cannot be (part of) an answer to the negated pattern.
```

Comment:
awww i like iff. It's a much stronger term than if :(

Reply:
Unless you mentally expand it to [https://en.wikipedia.org/wiki/iff]("if and only iff") as intended